sap-3-tier-marketplace-image-multi-sid-xscs-md/nestedtemplates/lb-DB-HANA.json

```diff
@@ -0,0 +1,164 @@
+            }
+        },
+        "nicBackAddressPoolsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the backaddress pools for the Network Interface creation"
+            }
+        },
+        "frontendIPConfigurationsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the frontend IP configurations for the Load Balancer creation"
+            }
+        },
+        "backendAddressPoolsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the backend address pools for the Load Balancer creation"
+            }
+        },
+        "loadBalancingRulesArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the fload balancing rules for the Load Balancer creation"
+            }
+        },
+        "probesArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the probes for the Load Balancer creation"
+            }
+        }
+    },
+    "variables": {
+        "lbPrefix": "[concat('lb', padLeft(parameters('dbInstanceNumber'), 2, '0'))]",
+        "lbrulePrefix": "[concat(variables('lbPrefix'), 'Rule')]",
+        "backendPoolDB": "[concat(variables('lbPrefix'), 'BackendPoolDB')]",
+        "loadBalancerFrontendDB": "[concat(variables('lbPrefix'), 'FrontendDB')]",
+        "lbProbePortDB": "[concat(variables('lbPrefix'), 'ProbePortDB')]",
+        "lbProbePortInternalDB": "[add(62500, parameters('dbInstanceNumber'))]",
+        "idleTimeoutInMinutesDB": 30,
+        "nicBackAddressPoolsResult": [
+            {
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', parameters('loadBalancerName')), '/backendAddressPools/', variables('backendPoolDB'))]"
+            }
+        ],
+        "frontendIPConfigurationsResult": [
+            {
+                "properties": {
+                    "subnet": {
+                        "id": "[parameters('subnetId')]"
+                    },
+                    "privateIPAllocationMethod": "Dynamic"
+                },
+                "name": "[variables('loadBalancerFrontendDB')]"
+            }
+        ],
+        "backendAddressPoolsResult": [
+            {
+                "name": "[variables('backendPoolDB')]"
+            }
+        ],
+        "loadBalancingRulesResult": [
+            {
+                "properties": {
+                    "frontendIPConfiguration": {
+                        "id": "[concat(resourceId('Microsoft.Network/loadBalancers', parameters('loadBalancerName')), '/frontendIpConfigurations/', variables('loadBalancerFrontendDB'))]"
```

Comment:
let's simplify these and just use resourceId

resourceId('Microsoft.Network/loadBalancers/frontendIpConfigurations', parameters('loadBalancerName')), variables('loadBalancerFrontendDB')